File: VolleyballApp/VolleyballApp.API/Models/Match.cs
using System;

namespace VolleyballApp.API.Models
{
    public class Match
    {
        public int Id { get; set; }
        public Team FirstTeam { get; set; }
        public Team SecondTeam { get; set; }
        public int ScoreId { get; set; }
        public Score Score { get; set; }
        public User Referee { get; set; }
        public bool IsRefereeInvited { get; set; }
        public Location Location { get; set; }
        public League League { get; set; }
    }
}

File: VolleyballApp/VolleyballApp-SPA/src/app/_models/match.ts
import { Score } from './score';
import { Team } from './team';
import { Location } from './location';
import { User } from './user';

export class Match {
    id: number;
    firstTeam: Team;
    secondTeam: Team;
    score?: Score;
    timeOfMatch?: Date;
    location?: Location;
    referee?: User;
    isRefereeInvited?: boolean;
}
